batavia/builtins/bytes.js

```diff
@@ -26,7 +27,19 @@ function bytes(args, kwargs) {
     } else if (args.length === 1) {
         var arg = args[0]
         if (arg === null) {
-            throw new exceptions.TypeError.$pyclass("'NoneType' object is not iterable")
+            switch (constants.BATAVIA_MAGIC) {
```

Comment:
It would be great if we could condense these cases down to a helper function, but I'm fine with that being a cleanup PR later.